tests/bor/bor_reorg_test.go

```diff
@@ -276,3 +298,131 @@ func TestValidatorWentOffline(t *testing.T) {
 	assert.Equal(t, authorVal1, nodes[0].AccountManager().Accounts()[0])
 
 }
+
+// TODO: Need to find a better name
+func TestForkWithTwoNodeNet(t *testing.T) {
```

Comment:
Can you parametrise this test so that we can try with different `sprintSize` and `blockTime`?
So that we can see how increasing those values impacts the network and get more precise info on when the fork happens? I suspect it's when `sprintSize > 64` but to confirm we need to have tests with multiple values (e.g. 32,64,128,256,512...)

Reply:
For example, you can use an approach like this one https://github.com/marcello33/go-core-dev-course/pull/1/files#diff-aaa393244cc727ef9aa3dfc6487ecd78336dcd26f93f67d9bb8579f7b6b6255fR60